provider/resource_user_attribution_basic_auth.go

```diff
@@ -35,6 +35,12 @@ func resourceUserAttributionBasicAuthRule() *schema.Resource {
 				Description: "url regex",
 				Optional:    true,
 			},
+			"disabled": {
```

Comment:
are we sure this field got added in gql for create flow?

Reply:
Its not in the create flow. But if we disable a rule from ui provider was not able to detect that change and do nothing so I have handled that part in read and update flow @murari54